tests/cases/classes/case

```diff
@@ -0,0 +1,16 @@
+---DESCRIPTION---
+Filename must match class name
+---FILENAME---
+foobar.php
+---CONTENTS---
+<?php
+
+namespace Vendor;
+
+class foobar
+{
+}
+
+---MESSAGES---
+5:1 Class name "foobar" is not in PascalCase format
```

Comment:
Example of something that can't be fixed automatically.

Reply:
Might be safer to include the name of the check rather than the text, as it’s more stable.

Reply:
Preference @giorgiosironi @stephenwf? Currently have to require `"squizlabs/php_codesniffer": "^3.3.1"` as one of the wordings changed (it reads `Squiz.Classes.ValidClassName.NotCamelCaps` if using the name of the check).

Reply:
I agree the class name in the context of this text should be closer to a published API. The user would still see the human message when running the tool or in other CI builds

Reply:
It may also output multiple messages for the same line with a non-deterministic order, but the current test cases don't go into that so not needed to cater for it.

Reply:
The class name may lose some detail about what the object being checked is (`Foo` class, `bar` method) but the line numbers make that clear anyway.